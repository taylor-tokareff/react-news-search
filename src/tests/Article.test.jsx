/**
 * @jest-environment jsdom
 */

import React from 'react';
import { render } from '@testing-library/react';
import Article from '../components/display/ArticleMain';

describe('Article component', () => {
  it('renders Article', () => {
    const { asFragment } = render(
      <Article
        title="Test title"
        description="Test description"
        author="Test author"
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});