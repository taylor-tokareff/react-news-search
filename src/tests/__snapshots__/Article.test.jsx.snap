// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <div>
    <p>
      Test title
    </p>
    <p>
      Test author
    </p>
    <p>
      Test description
    </p>
  </div>
</DocumentFragment>
`;
